### DISCUSSION TITLE
Uncertainty Parameters

### DISCUSSION THREAD

**Discussion by rnugent3** [HydrologicEngineeringCenter/HEC-FDA#319] [Q&A] [answered]
I am thinking about separating out first floor elevation uncertainty, foundation height uncertainty, and terrain elevation uncertainty into separate uncertainty parameters for a given occupancy type. 

Often, first floor elevation uncertainty is often confused with foundation height uncertainty, ignoring terrain elevation uncertainty. Presenting the three parameters as unique where either first floor elevation uncertainty is specified or both foundation height uncertainty and terrain elevation uncertainty are specified should make the difference clear to the user, improving the chance that these parameters will be specified correctly. This seems like basic UX - a workflow that helps the user get where they need to be, minimizing the potential for error. 

What's more, the different uncertainty distributions lend to different distributional assumptions. That is, foundation height uncertainty is naturally described by a triangular distribution, but terrain elevation uncertainty is naturally described by a Normal distribution. When first floor elevation uncertainty is confused with foundation height uncertainty and a normal distribution is used to represent the uncertainty (as is often the practice), we end up with non-sensical uncertainty distributions. 

This would not require we modify our occupancy type import procedure, only the occupancy type editor and database. Having not yet started the aggregated stage damage code, we still have a lot of "free" flexibility in how the stage-damage compute is done. 

Team, any thoughts?

**Comment by rnugent3** [comment, marked as answer]
Terrain elevation uncertainty need not be found in an occupancy type. Also, this could be held off for 2.1 if needed, but I think this would probably a small thing with big benefits

**Reply by HenryGeorgist** [reply]
How much does this contribute to changing the outcome of a decision on a project?

**Reply by rnugent3** [reply]
Good question. I don't think we know. I am thinking about the potential for non-sensical foundation heights. I don't know what the impact could be. Adding this to our "projects to fund" list.

**Reply by rnugent3** [reply]
The more I think about this, the more complications come to mind. I think we should stick to our current approach, icky as it may be, and do some research to inform how we might improve
